webapp-django/crashstats/crashstats/models.py

```diff
@@ -107,6 +107,18 @@ class Meta:
         unique_together = ('vendor_hex', 'adapter_hex')
 
 
+class Platform(models.Model):
+    """Lookup table for platforms"""
+    name = models.CharField(
+        max_length=20, blank=False, null=False,
```

Comment:
This field has four values presently: "Windows", "Mac OS X", "Linux", and "Unknown". None of these are 20 characters, but I figured I'd give it a bit of room.